iceoryx_hoofs/include/iceoryx_hoofs/internal/relocatable_pointer/relative_pointer_data.hpp

```diff
@@ -57,25 +57,32 @@ class RelativePointerData
     bool isLogicalNullptr() const noexcept;
 
     /// @note the maximum number of available ids
-    static constexpr id_t ID_RANGE{std::numeric_limits<id_t>::max()};
+    // AXIVION Next Line AutosarC++19_03-M11.0.1 : Needs to be verified outside of the class
```

Comment:
I would actually vote to deactivate this rule else we have 100 suppressions in hoofs alone.